pages/code-of-conduct.tsx

```diff
@@ -10,89 +10,160 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
-      who will be identifiable by the branded DDD Perth t-shirts they'll be wearing.
+      If a participant engages in harassing behaviour, event organisers retain the right to take any actions to keep the
+      event a welcoming environment for all participants. This includes warning the offender or expulsion from the
+      conference with no refund.
     </p>
 
-    <h2>The Quick Version</h2>
     <p>
-      Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of
-      gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race,
-      ethnicity, or religion (or lack thereof). We do not tolerate harassment of conference participants in any form.
-      Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties,
-      Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from
-      the conference <em>without a refund</em> at the discretion of the conference organisers.
+      Event organisers may take action to redress anything designed to, or with the clear impact of, disrupting the
+      event or making the environment hostile for any participants. We expect participants to follow these rules at all
+      event venues, event-related social activities as well as social media.
     </p>
 
-    <h2>The Less Quick Version</h2>
+    <h2>Reporting</h2>
+
     <p>
-      Harassment includes offensive verbal comments related to gender, gender identity and expression, age, sexual
-      orientation, disability, physical appearance, body size, race, ethnicity, religion, sexual images in public
-      spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of
-      talks or other events, inappropriate physical contact, and unwelcome sexual attention.
+      If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. Conference
+      staff can be identified by event branded t-shirts. Harassment and other Code of Conduct violations reduce the
+      value of our event for everyone. We want you to be happy at our event. People like you make our event a better
+      place. You can make a report either personally or anonymously.
     </p>
-    <p>Participants asked to stop any harassing behavior are expected to comply immediately.</p>
+
+    <h3>Anonymous report</h3>
+
+    <p>
+      You can <a href={props.pageMetadata.conference.AnonymousReportFormUrl}>make an anonymous report here</a>. We can't
+      follow up an anonymous report with you directly, but we will fully investigate it and take whatever action is
+      necessary to prevent a recurrence.
+    </p>
+
+    <h3>Personal report</h3>
+
+    <p>You can make a personal report by:</p>
+
+    <ul>
+      <li>Contacting a staff member, identified by event branded t-shirts</li>
```

Comment:
Whatever we put above should also be put here, might make sense as a variable in the Conference class